components/DocsLayout.js

```diff
@@ -44,6 +44,7 @@ class DocsLayout extends Component {
           title={`styled-components${title ? `: ${title}` : ''}`}
           description={description}>
             <meta name="robots" content="noodp" />
+            <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />
```

Comment:
I'd either vendor it into our code or style it manually, I think to keep the requests low. We're already at a high number of requests due to Google

Reply:
Sure! I wanted to do that, but I thought maybe styles are huge to include. So I'll do this with `injectGlobal` or I should add a style tag inside `<Head>` by hand?

Reply:
So, the difference will be small especially on HTTP 2, but this is a small CSS file after all. I can verify what it actually contains and see.

Reply:
btw About Google, we should probably replace that with something else, maybe Heap. Google Analytics is not accurate and also it's so heavy.

Reply:
We spent months getting GA getting set up correctly, I'm not switching to anything different now. I highly, highly doubt anything else is more accurate.

Reply:
I have proof!

Reply:
https://medium.com/@timmycarbone/google-analytics-modifies-your-data-24d4d6366210

Reply:
https://heapanalytics.com/compare/heap-vs-google-analytics